packages/components/badge/src/Badge.tsx

```diff
@@ -7,16 +7,17 @@ import { getBadgeStyles } from './getBadgeStyles';
 
 export interface BadgeProps extends CommonProps {
   /**
-   * property to set size of the component
+   * Sets the size of the component
    * @default default
    */
   size?: BadgeSize;
   /**
-   * prop, used for setting variation of the component
+   * Determines the variation of the component
    * @default primary
    */
   variant?: BadgeVariant;
-  children: React.ReactNode;
+
+  children: React.ReactText;
```

Comment:
this changed from ReactNode to ReactText - is it always just text?

Reply:
As far as I know - yes. It's better to narrow down the type here to prevent crazy usage of the Badge component.

Reply:
yeah, I get it - was just thinking whether it ever happens that we, for example, throw an Icon in there...

Reply:
yeah, let's verify with folks: @burakukula @gui-santos @mshaaban0

Reply:
I believe it should also accept icons? we use the clock icon for scheduled entities, so maybe ReactNode is the way to go here

Reply:
okay, rolling it back to React.ReactNode